assets/js/projectDet.js

```diff
@@ -0,0 +1,160 @@
+const projectHistory = [
```

Comment:
- [Optional] On desktop size, the pop-up is not in the center of the screen. There's some space on the right side of the screen. Please kindly fix the minor issue. You can consider also reducing the size of the image so that the pop-up is not scrollable.

Reply:
Thank you for the review @FrederickMih . Am going to center this right away